fluent-tests/src/test/java/com/azure/mgmttest/RuntimeTests.java

```diff
@@ -117,6 +117,7 @@ public void testManagementClient() {
     }
 
     @Test
+    @Disabled("Enable when deserialization for error subclasses are fixed.")
```

Comment:
Stream-style for error subclass seems to have issue. 
Created: https://github.com/Azure/autorest.java/issues/2826

Reply:
Just use `steam-style-serialization=false` for this case. Don't disable the test.

Reply:
updated